app/views/shared/_navbar.html.erb

```diff
@@ -4,18 +4,21 @@
     <img src="<%= image_path 'logo.png', crop: :fill %>" alt="logo">
   </a>
 
-  <form>
```

Comment:
Removed the search bar. We will add this in after. We have this as a button for right now.